packages/o-spreadsheet-engine/src/formulas/parser.ts

```diff
@@ -91,6 +91,11 @@ export interface ASTSymbol extends ASTBase {
   value: string;
 }
 
+export interface ASTArray extends ASTBase {
+  type: "ARRAY";
+  rows: AST[][];
```

Comment:
I would name it `value` to avoid all the mlsqjfqsf required because all AST no longer has a `value` property
```suggestion
  value: AST[][];
```